plugins/base/src/main/kotlin/renderers/DefaultRenderer.kt

```diff
@@ -170,6 +170,23 @@ abstract class DefaultRenderer<T>(
             renderPages(newRoot)
         }
     }
+
+    protected fun ContentDivergentGroup.groupDivergentInstances(
```

Comment:
Is there no option to make this simpler?
Maybe implement a custom type instead of using `Map<Pair<String, String>, List<Pair<ContentDivergentInstance, DokkaSourceSet>>>`

Reply:
I can do, I just ripped it from here, hadn't ever thought about tough signature 
https://github.com/Kotlin/dokka/blob/dev-0.11.0/plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt#L226